.github/workflows/dist.yml

```
@@ -1,5 +1,6 @@
 name: Dist
 on:
+  dispatch_workflow:
```

actionlint (1.7.8)

unknown Webhook event "dispatch_workflow". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)